porcupine/plugins/filemanager.py

```diff
@@ -122,6 +169,38 @@ def rename(old_path: Path) -> None:
         tab.path = new_path / tab.path.relative_to(old_path)
 
 
+def paste(new_path: Path) -> None:
+    paste_here(new_path.parent)
+
+
+def paste_here(new_path: Path) -> None:
+    assert is_copy_path_valid()
+    assert new_path.is_dir()
```

Comment:
Unfortunately mypy (the type checker) isn't clever enough to see how `assert is_copy_path_valid()` ensures thjat `copy_path` can't be `None`. That currently causes a mypy error. The rest of the checks are not needed, because theu don't affect type checking and they are already handled by `can_paste_here(new_path)`.

```suggestion
    assert copy_path is not None
```

Reply:
I'm sorry for these repeated problems. I've never used mypy before. Should check smth about it btw.

Reply:
It's ok, I'm here to help :)

Reply:
I don't know why **black check** wasn't passed. I ran it and now that I'm trying again it says that there's nothing to change.